docs/src/Video.doc.js

```diff
@@ -0,0 +1,345 @@
+      },
+      {
+        name: 'accessibilityMinimizeLabel',
+        type: 'string',
+        description:
+          'Accessibility label for the fullscreen minimize button if controls are shown',
+      },
+      {
+        name: 'accessibilityMuteLabel',
+        type: 'string',
+        description:
+          'Accessibility label for the mute button if controls are shown',
+      },
+      {
+        name: 'accessibilityPauseLabel',
+        type: 'string',
+        description:
+          'Accessibility label for the pause button if controls are shown',
+      },
+      {
+        name: 'accessibilityPlayLabel',
+        type: 'string',
+        description:
+          'Accessibility label for the play button if controls are shown',
+      },
+      {
+        name: 'accessibilityUnmuteLabel',
+        type: 'string',
+        description:
+          'Accessibility label for the unmute button if controls are shown',
+      },
+      {
+        name: 'captions',
+        type: 'string',
+        description: 'The URL of the captions track for the video (.vtt file)',
+        required: true,
+      },
+      {
+        name: 'controls',
+        type: 'boolean',
+        description: 'Show the video player controls',
+      },
+      {
+        name: 'loop',
+        type: 'boolean',
+        description: 'The video will start playing over again when finished',
+      },
+      {
+        name: 'onDurationChange',
+        type:
+          '({ event: SyntheticEvent<HTMLVideoElement>, duration: number }) => void',
+        description:
+          'Sent when the metadata has loaded or changed, indicating a change in duration',
+      },
+      {
+        name: 'onFullscreenChange',
+        type: '({ fullscreen: boolean }) => void',
+        description: 'Sent when the video full screen status changes',
+      },
+      {
+        name: 'onPlay',
+        type: '() => void',
+        description:
+          'Sent when playback of the media starts after having been paused',
+      },
+      {
+        name: 'onPause',
+        type: '() => void',
+        description: 'Sent when playback is paused',
+      },
+      {
+        name: 'onTimeUpdate',
+        type:
+          '({ event: SyntheticEvent<HTMLVideoElement>, currentTime: number }) => void',
+        description:
+          "Sent when the time indicated by the element's currentTime attribute has changed",
+      },
+      {
+        name: 'onVolumeChange',
+        type: '({ volume: number }) => void',
+        description: 'Sent when the audio volume changes',
+      },
+      {
+        name: 'playing',
+        type: 'boolean',
+        description: 'Specifies whether the video should play or not',
+        defaultValue: false,
+      },
+      {
+        name: 'playsInline',
+        type: 'boolean',
+        description:
+          'Indicates that the video is to be played "inline" where supported',
```

Comment:
I'm not sure sure what this means from this description - what actually changes with it being inline?